Gems/Atom/Tools/AtomToolsFramework/Code/Include/AtomToolsFramework/Viewport/ModularViewportCameraControllerRequestBus.h

```diff
@@ -29,25 +29,31 @@ namespace AtomToolsFramework
         static const AZ::EBusAddressPolicy AddressPolicy = AZ::EBusAddressPolicy::ById;
         static const AZ::EBusHandlerPolicy HandlerPolicy = AZ::EBusHandlerPolicy::Single;
 
-        //! Begin a smooth transition of the camera to the requested transform.
+        //! Begins a smooth transition of the camera to the requested transform.
         //! @param worldFromLocal The transform of where the camera should end up.
         //! @return Returns true if the call began an interpolation and false otherwise. Calls to InterpolateToTransform
         //! will have no effect if an interpolation is currently in progress.
         virtual bool InterpolateToTransform(const AZ::Transform& worldFromLocal) = 0;
-
         //! Returns if the camera is currently interpolating to a new transform.
         virtual bool IsInterpolating() const = 0;
-
-        //! Start tracking a transform.
+        //! Starts tracking a transform.
         //! Store the current camera transform and move to the next camera transform.
         virtual void StartTrackingTransform(const AZ::Transform& worldFromLocal) = 0;
-
-        //! Stop tracking the set transform.
+        //! Stops tracking the set transform.
         //! The previously stored camera transform is restored.
         virtual void StopTrackingTransform() = 0;
-
-        //! Return if the tracking transform is set.
+        //! Returns if the tracking transform is set.
         virtual bool IsTrackingTransform() const = 0;
+        //! Sets the current camera pivot, moving the camera offset with it (the camera appears
+        //! to follow the pivot, staying the same distance away from it).

```

Comment:
Nit: Mismatching commenting style "Sets/Stops" with an s vs. "Set/Store"

Reply:
I've updated them now to be consistent? I want the comment to describe what the function does, not what it is doing right now (comments like that make sense inline in a function body, not as part of the API docs) - I was just fixing that here, what do you mean I should change it to?

Reply:
Oooh I see on line 53, will fix 👍 (sorry diff here missed it)